libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.html

```diff
@@ -34,8 +34,8 @@
             [aria-label]="checkboxLabel(member)"
             [checked]="selection.isSelected(member)"
             attr.data-cy="{{member.user.firstName | lowercase}}-checkbox"
-            [disabled]="member.membershipType === 'INDIRECT'"
-            [matTooltip]="member.membershipType === 'INDIRECT' ? ('MEMBERS_LIST.CHECKBOX_TOOLTIP' | translate):''"
+            [disabled]="member.membershipType === 'INDIRECT' || !(member | memberLifecycleAlterable)"
```

Comment:
```suggestion
            member | memberListCheckboxDisabled"
```
and in that pipe you will make decision to disable the checkbox if the member is indirect or if the attribute is lifecycle alterable is false